content/reference/api/hub/latest-changelog.md

```diff
@@ -11,6 +11,13 @@ toc_max: 2
 Here you can learn about the latest changes, new features, bug fixes, and known
 issues for Docker Service APIs.
 
+## 2025-06-27
+
+### New
+
+- Add [List repositories](/reference/api/hub/latest/#tag/repositories/operation/listNamespaceRepositories) endpoints for a given `namespace`.
+- Deprecate undocumented endpoint `GET /v2/repositories/{namespace}` replaced by [List repositories](/reference/api/hub/latest/#tag/repositories/operation/listNamespaceRepositories).
```

Comment:
Not sure what we did in the past, but perhaps for visibility (although there’s only 2 things in the bullets), we could use ### New for the new endpoint, and a ### Deprecations (or similar) for what’s deprecated. That could also add a link to the “deprecated” page (refer to “deprecated” for a list of deprecated API endpoints / API features)

Reply:
Probably also want to add a mention in deprecated; not sure if we have a good format for that already, but there’s some existing things that were deprecated; https://github.com/docker/docs/blob/main/content/reference/api/hub/deprecated.md

We should probably start thinking of some more formalized deprecation document (what’s our policy, how announced, etc), although in most cases actually removing endpoints / features from the API may be more complicated, or at least, not without a major version bump of the API and ultimately fully dropping support for prior major versions; https://docs.docker.com/engine/deprecated/